src/plugin.js

```diff
@@ -190,6 +197,7 @@ const onPlayerReady = (player, options) => {
         <h2 class="vjs-errors-headline">${this.localize(error.headline)}</h2>
           <div><b>${this.localize('Error Code')}</b>: ${(error.type || error.code)}</div>
           ${details}
+          <i><span class="vjs-errors-flashmessage">${flashMessage}</span></i>
```

Comment:
I don't think this element should be included if `flashMessage` is empty. Rather than creating a new variable, what about appending it to `details`?

Also, on the HTML side, I think the element nesting ought to be flipped (i.e. `<span ...><i>`) or even the `<i>` eliminated and the style added through CSS.